src/PiWeb.Import.Sdk/Modules/ImportFormat/ImportGroupExtensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss Industrielle Messtechnik GmbH        */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2024                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#endregion
+
+namespace Zeiss.PiWeb.Import.Sdk.Modules.ImportFormat;
+
+#region usings
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+#endregion
+
+/// <summary>
+/// Provides methods to work with <see cref="IImportGroup"/> more easily.
+/// </summary>
+public static class ImportGroupExtensions
+{
+    #region methods
+
+    /// <summary>
+    /// Checks whether <paramref name="importGroup"/>'s <see cref="IImportGroup.PrimaryFile"/> has the requested <paramref name="extension"/>.
+    /// </summary>
+    /// <param name="importGroup">The <see cref="IImportGroup.PrimaryFile"/> to check.</param>
+    /// <param name="extension">The file extension (including the period ".").</param>
+    /// <returns><c>true</c> if the <see cref="IImportGroup.PrimaryFile"/> has the extension; otherwise <c>false</c>.</returns>
+    public static bool HasExtension(this IImportGroup importGroup, string extension)
+    {
+        if (!extension.StartsWith('.'))
+            extension = '.' + extension;
+
+        return importGroup.PrimaryFile.Name.EndsWith(extension, StringComparison.OrdinalIgnoreCase);
```

Comment:
I'm not sure whether we need those extension methods...
But we should at least use the FileSourceExtension here instead of implementing the comparison again:
`return importGroup.PrimaryFile.HasExtension(extension);`

Reply:
You are right